### PR TITLE
Cqm integration valuesets

### PR DESCRIPTION
**Pull request by lukoktonos**
Remove cql_calculator and only use cqm_calculator

Instead of using `bonnie.valueSetsByMeasureId`, set thorax measure `cqmValueSets` per measure. Update tests to load them in from `value_sets.json` files.

List of updated specs for running in teaspoon:
- spec/javascripts/cql_calculator_spec.js.coffee (expected fails)
- spec/javascripts/helper_specs/compare_version_spec.js.coffee
- spec/javascripts/helper_specs/cql_measure_helpers_spec.js.coffee
- spec/javascripts/helper_specs/valueset_helpers_spec.js.coffee
- spec/javascripts/integration/composite_measures_spec.js.coffee (expected fails)
- spec/javascripts/integration/navigation_spec.js.coffee
- spec/javascripts/integration/population_state_spec.js.coffee (expected fail)
- spec/javascripts/models/coverage_spec.js.coffee (expected fail)
- spec/javascripts/models/expected_value_spec.js.coffee (expected fails)
- spec/javascripts/models/measure_spec.js.coffee (expected fails)
- spec/javascripts/models/result_spec.js.coffee (expected fails)
- spec/javascripts/production_tests/production_patient_builder_spec.js.coffee (expected fails)
- spec/javascripts/views/cql_coloring_spec.js.coffee (expected fails)
- spec/javascripts/views/cql_logic_view_spec.js.coffee (expected fail)
- spec/javascripts/views/cql_truncated_statement_view_spec.js.coffee (expected fail)
- spec/javascripts/views/import_measure_view_spec.js.coffee
- spec/javascripts/views/measure_view_spec.js.coffee (fails, update value sets tests?)
- spec/javascripts/views/measures_view_spec.js.coffee
- spec/javascripts/views/patient_builder_spec.js.coffee (expected fails)
- spec/javascripts/views/population_calculation_view_spec.js.coffee
 
Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] This PR is into the correct branch.
- [x] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1963
- [x] JIRA ticket links to this PR
- [x] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [x] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [x] Test fixtures updated and documented as necessary ( see [internal wiki](https://gitlab.mitre.org/bonnie/internal-documentation/wikis/testing#test-fixtures) )
- [x] Code coverage has not gone down and all code touched or added is covered. 
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here: 
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here: 
- [x] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [x] JIRA test links:
- [x] Justification for using JIRA tests:
- [x] JIRA tests have been added to sprint

**Reviewer 1:**

Name:
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [x] ~JIRA tests have been run and pass~
- [x] ~You agree with the justification for use of JIRA tests or have provided input on why you disagree~

### CODE REVIEW

File: app/controllers/measures_controller.rb [hunk 1]
```diff
@@ -19,14 +22,9 @@ def show
   end
 
   def value_sets
```
Comment (jbradl11) [inline app/controllers/measures_controller.rb]: If we are sending the value-sets along with the measure in `show` is this function used anymore?

File: app/views/measures/value_sets.js.erb [hunk 2]
```diff
@@ -1 +0,0 @@
-bonnie.valueSetsByMeasureId = <%= @value_sets_final.html_safe %>;
```
Comment (jbradl11) [inline app/views/measures/value_sets.js.erb:1]: Based on this being removed, I think that answers my questions about the `value_sets` function. Can you delete it in this PR?

**Review by jbradl11 — APPROVED** [review]
